public/botchat.css

```diff
@@ -1,403 +1,412 @@
-    }
-
-    .wc-carousel button.scroll svg {
-        fill: #fff;
-    }
-    
-    .wc-carousel button.scroll.previous {
-        left: -16px;
-    }
-
-    .wc-carousel button.scroll.next {
-        right: -16px;
-    }
-
-    .wc-carousel-scroll-outer {
-        /* allow horizontal scrolling but hide the scrollbar */
-        overflow: hidden;
-    }
-
-    .wc-carousel-scroll {
-        /* allow horizontal scrolling but hide the scrollbar */
-        overflow-x: scroll;
-        overflow-y: hidden;
-    }
-
-    .wc-carousel-scroll > ul {
-        margin-left: -4px;
-        white-space: nowrap;
-    }
-
-    .wc-carousel-scroll > ul > li {
-        display: inline-block;
-        padding: 0 4px;
-        vertical-align: top;
-        white-space: nowrap;
-    }
-
-    .wc-carousel-scroll > ul > li:last-child {
-        padding-right: 0;
-    }
-
-    .wc-carousel li p {
-        min-height: 4em;
-        white-space: normal;
-    }
-
-/* from me */
-
-    .wc-message-from-me {
-        float: right;
-        margin-right: 12px;
-    }
-
-    .wc-message-from-me .wc-message-from {
-        text-align: right;
-    }
-
-    .wc-message-from-me .wc-message-content {
-        background-color: #808c95;
-        color: #fff;
-    }
-
-    .wc-message-from-me svg.wc-message-callout path {
-        fill: #808c95;
-    }
-
-    .wc-message-from-me svg.wc-message-callout path.point-left {
-        display: none;
-    }
-
-    .wc-message-from-me svg.wc-message-callout {
-        right: -10px;
-    }
-
-/* from bot */
-
-    .wc-message-from-bot {
-        float: left;
-        margin-left: 12px;
-    }
-
-    .wc-message-from-bot .wc-message-content {
-        background-color: #eceff1;
-        color: #000;
-    }
-
-    .wc-message-from-bot svg.wc-message-callout path {
-        fill: #eceff1;
-    }
-
-    .wc-message-from-bot svg.wc-message-callout path.point-right {
-        display: none;
-    }
-
-    .wc-message-from-bot svg.wc-message-callout {
-        left: -12px;
-    }
-
-/* console */
-
-    .wc-console > * {
-        position: absolute;
-        top: 0;
-        vertical-align: middle;
-    }
-
-    .wc-console label {
-        cursor: pointer;
-        display: inline-block;
-        height: 40px;
-    }
-
-    .wc-console svg {
-        fill: #8A8A8A;
-        margin: 11px;
-    }
-
-    .wc-console textarea, .wc-console input[type=text] {
-        border: none;
-        height: 100%;
-        outline: none;
-        padding: 0;
-        resize: none;
-        width: 100%;
-    }
-
-    .wc-upload input {
-        display: none;
-    }
-
-    .wc-textbox {
-        bottom: 0;
-        left: 48px;
-        right: 49px;
-    }
-
-    .wc-send {
-        right: 0;
-    }
-
-/* animation */
-
-    .wc-animate-scroll {
-        transition: left .8s ease;
-    }
-
-    .wc-animate-scroll-rapid {
-        transition: left .4s ease;
-    }
-
-    .wc-animate-scroll-near {
-        transition: left .5s ease-in-out;
-    }
-
+﻿/* reset */

```

Comment:
for some reason this entire file is showing as "changed". Maybe an indentation thing? I'd really like just to see the lines that are different.